fetch-api/src/services/errors.js

```diff
@@ -0,0 +1,11 @@
+import _ from 'lodash';
+
+export function getError(action) {
```

Comment:
in fetch api we work with responses from fetch function. There is no actions, should we have seperate error function for fetch-api and redux-api because of that?